docs/cli.md

```diff
@@ -138,10 +138,13 @@ Parameters:
 
 |Name|Description|
 |----|-----------|
-|targetType|The type of item to mock|
+|targetType|The type of item to mock (plugin, command)|
 |targetName|The item to mock|
 |targetValue|The mocked value|
 
+example: `oc mock plugin foo bar` adds a static mock for plugin `foo` to return value `bar`.
+example: `oc mock plugin foo ./bar.js` adds a dynamic mock using file `./bar.js`
+
 ##preview
```

Comment:
The `cli.md` is generated during publish by a grunt task starting from [the cli commands](https://github.com/opentable/oc/blob/master/cli/commands.js) - you need to make this change there in order to get this propagated here - this will make the cli help up-to-date too. ([here](https://github.com/opentable/oc/blob/master/Gruntfile.js#L67-L74) is the grunt task)